### PR TITLE
Real homework

### PR DESCRIPTION
**Pull request by okggff**
I finished my homework for problems **#6 (factoring)** and **#13 (triangle inequality theorem)**. let me know if you have any problems. I will continue the homework tomorrow!  My **finished** homework is "[Commit from on Mon, Dec 16, 2024 8:52:20 PM](https://github.com/JaehoonSong12/ydjs_academic/pull/23/commits/85132c8edab19135e7f93901d0335dd115887df9)"

### CODE REVIEW

File: docs/JAYDEN.md [hunk 1]
```diff
@@ -1,40 +1,74 @@
...
+<a id="readme-top"></a>
+
+
+
 
 
 
 ## Table of Contents
 - [Math: SAT](#math-sat)
+- [📐 Triangle Inequality Theorem](#-triangle-inequality-theorem)
```
Comment (JaehoonSong12) [inline docs/JAYDEN.md:46]: do not use these icons, it makes the content hard to read.

File: docs/JAYDEN.md [hunk 2]
```diff
@@ -1,40 +1,74 @@
...
+  - [🎯 What is the Triangle Inequality Theorem?](#-what-is-the-triangle-inequality-theorem)
+    - [🔢 In Math Terms:](#-in-math-terms)
+  - [📝 Why is this True?](#-why-is-this-true)
```
Comment (JaehoonSong12) [inline docs/JAYDEN.md:49]: Titles are not aligned by their importance level.

Why is "Triangle Inequality Theorem" heading level 1? it does not have consistency with the other topic, "absolute value".

File: docs/JAYDEN.md [hunk 3]
```diff
@@ -1,40 +1,74 @@
...
+
+
+
 
 
 
 ## Table of Contents
 - [Math: SAT](#math-sat)
+- [📐 Triangle Inequality Theorem](#-triangle-inequality-theorem)
+  - [🎯 What is the Triangle Inequality Theorem?](#-what-is-the-triangle-inequality-theorem)
```
Comment (JaehoonSong12) [inline docs/JAYDEN.md:47]: For headings, use short, but descriptive nouns, not sentences.

Headers are supposed to be less then 3 words, sometimes with parentheses. ()

File: docs/JAYDEN.md [hunk 4]
```diff
@@ -66,81 +113,426 @@
...
+
 # Math: SAT
 
 
 
 
 
+
+# 📐 Triangle Inequality Theorem
+![Triangle Inequality Theorem](./images/Triangle-Inequality-Theorm.webp)
```
Comment (JaehoonSong12) [inline docs/JAYDEN.md:128]: This image is not conceptual, it is not a presentation.

Be more practical.

File: docs/JAYDEN.md [hunk 5]
```diff
@@ -66,81 +113,426 @@
...
+
+The **Triangle Inequality Theorem** states that:  
+
+
+> **The sum of the lengths of any two sides of a triangle must be *greater* than the length of the third side.**
+
+
+### 🔢 In Math Terms:  
+For any triangle with sides **a**, **b**, and **c**:  
+- ✅ **a + b > c**
```
Comment (JaehoonSong12) [inline docs/JAYDEN.md:142]: Too many boldfaced words, boldface must be used a few times to highlight important contexts, not everywhere.

**Review by JaehoonSong12 — CHANGES_REQUESTED** [review]
Review has been done, see the `files changed` section to see the details.

Modify majority of the context, you are supposed to write-up feedback short but practical, not textbook-like paragraphs.

**Comment by okggff** [comment]
I'm can't git02-branch+push.sh the fixed homework

**Comment by okggff** [comment]
I  git02-branch+push.sh my homework, but I can't remember next step.

**Comment by okggff** [comment]
and I can't see my fixed homework anywhere in github

**Comment by JaehoonSong12** [comment]
Things to remember
#1: do not rush, if you cannot figure something out, write-up about the situation with a single reply
#2: do not delete the branches that contains all your work. Seat back and wait for my response.

I will revert the branch deletion, once you pull everything, manually type in the following command to work on your feedback!
`git checkout Real_Homework`

**Comment by okggff** [comment]
I pulled and `git checkout Real_Homework`. Should I create a new branch to push my homework after it's finished?

**Comment by JaehoonSong12** [comment]
Up to you! You are all set at the point!

**Comment by okggff** [comment]
Alright!, and since I can't bring my desktop to Korea, I can't do my desktop related homework, so I can only do paper homework.

**Pull request closed without merging**